### ISSUE TITLE
Rest parameters on >0.4.3 and <0.6.1

### ISSUE THREAD

**Issue by ilikejames** [re-rxjs/react-rxjs#156]
This is fixed in `0.6.1` so just looking for advice on this, and whether a fix can be patched into `0.5.x`
Affects versions >0.4.3 <0.6.1

```ts
const [useRequiresTheArray] = bind((s: string, a: string[]) => {
  return of(a.join(s));
});
```

is fine, but

```ts
const [useRequiresTheArray] = bind((s: string, ...a: string[]) => {
  return of(a.join(s));
});
```
```text
Warning: The final argument passed to useEffect changed size between renders. The order and size of this array must remain constant.
```

[Example Code](https://codesandbox.io/s/upbeat-driscoll-d8knk?file=/src/App.js)

**Comment by voliva** [comment]
Patched on 0.5.2

**Comment by ilikejames** [comment]
Brilliant!
